meta-intel-edison-distro/recipes-support/oobe/oobe_git.bb

```diff
@@ -14,6 +14,7 @@ RDEPENDS_${PN} = "python3-core python3-bottle"
 
 do_install() {
    install -d ${D}${libdir}/edison_config_tools
+   install -d ${D}/var/lib/edison_config_tools
```

Comment:
> Didn't get why you need to duplicate installation of edison_config_tools folder. Does ${libdir} not work correctly? Where does it point to?

`${libdir}` points `/usr/lib`, so `/usr/lib/edison_config_tools/public/` is where the web server template pages are installed. But for some reason /var/lib/edison_config_tools needs to exists otherwise the CLI is broken. python mysteries...

Reply:
Exactly, the _cli_ creates some temporary file there: https://github.com/edison-fw/edison-oobe/blob/07512dfcb97d77c57adb024c6ca3473abbae0229/src/configure_edison#L798